contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -211,8 +222,20 @@ contract StakeManager is
         token = IERC20(_token);
     }
 
+    // @note
+    function setLegacyToken(address _token) public onlyGovernance {
+        require(_token != address(0x0));
+        tokenLegacy = IERC20(_token);
+    }
+
+    // @note
+    function setMigration(address _migration) public onlyGovernance {
```

Comment:
nit: when would we need to re-set legacyToken & migration?

Reply:
I thought as a safety precaution.
If you feel it's unneeded I'll remove them.

Reply:
i do feel it's unneeded since migration is behind a proxy & legacyToken by definition shouldn't be changed, rming saves us some bytecode as well 😉